src/resources/postcss/views/day/_event.pcss

```diff
@@ -28,6 +28,21 @@
 		}
 	}
 
+	.tribe-events-calendar-day__event--featured {
+
+		.tribe-events-calendar-day__event-details {
+
+			&:before {
+				background: var(--color-accent-primary);
+				content: '';
+				height: 100%;
+				left: var(--grid-gutter-small-half-negative);
```

Comment:
shouldn't this have different values for mobile and desktop (since the gutter changes)?